airbyte-integrations/connectors/destination-snowflake/README.md

```diff
@@ -8,23 +8,76 @@
 1. Create a file at `secrets/config.json` with the following format:
 ```
 {
-  "host": "ACCOUNT.REGION.PROVIDER.snowflakecomputing.com",
+  "host": "testhost.snowflakecomputing.com",
   "role": "AIRBYTE_ROLE",
   "warehouse": "AIRBYTE_WAREHOUSE",
   "database": "AIRBYTE_DATABASE",
   "schema": "AIRBYTE_SCHEMA",
   "username": "AIRBYTE_USER",
-  "password": "SOMEPASSWORD"
+  "credentials": {
+    "password": "test"
+  }
 }
 ```
 
 ## For Airbyte employees
 Put the contents of the `Snowflake Integration Test Config` secret on LastPass under the `Engineering` folder into `secrets/config.json` to be able to run integration tests locally.
 
 1. Put the contents of the `destination snowflake - insert test creds` LastPass secret into `secrets/insert_config.json`.
```

Comment:
the lastpass creds are still using the old values - are you planning to update them, or should this list be modified to reference the GSM entries?

(hackweek thought: can fetching these secrets be done via script)

Reply:
I'll do a future update here for dev credentials. One prong of the current work is to separate out the CI world fro the dev world.

Reply:
hack day delivered?! https://github.com/airbytehq/airbyte/blob/master/tools/ci_credentials/ci_credentials/secrets_loader.py

Reply:
whoa, how did I not hear about this?

do we not want to keep CI and dev secrets identical? (might be easier to move this discussion out of github comments tbh)